contracts/core/EntryPoint.sol

```diff
@@ -441,6 +443,11 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuardT
     ) internal {
         if (initCode.length != 0) {
             address sender = opInfo.mUserOp.sender;
+            if ( _isEip7702InitCode(initCode) ) {
+                //already validated it is an EIP-7702 delegate (and hence, already has code)
+                senderCreator().initEip7702Sender(sender, initCode[20:]);
```

Comment:
I was under impression that you wanted to call `initEip7702Sender` only if `initCode` contains some data in addition to `0xff0100`, but this code seems to call it every time.